lib/vdsm/storage/lvm.py

```diff
@@ -1318,7 +1318,7 @@ def getVGbyUUID(vgUUID):
             log.debug("%s", e, exc_info=True)
             continue
     # If not cry loudly
-    raise se.VolumeGroupDoesNotExist("vg_uuid: %s" % vgUUID)
+    raise se.VolumeGroupDoesNotExist(UNKNOWN, f"Not found vg_uuid {vgUUID}")
```

Comment:
Looks like we are abusing se.VolumeGroupDoesNotExist here. Since engine is using this
error, we probably need to keep using it for both case for looking up VG by name and uuid, so need to make the error more flexible, so we can use it for both cases:

    se.VolumeGroupDoesNotExist(vg_uuid="vg-uuid")
    se.VolumeGroupDoesNotExist(vg_name="vg-name")
    se.VolumeGroupDoesNotExist(vg_name="vg-name", error=e)